lib/simplefootball_web_web/controllers/competition_controller.ex

```diff
@@ -0,0 +1,48 @@
+defmodule SimplefootballWebWeb.CompetitionController do
+  use SimplefootballWebWeb, :controller
+
+  alias SimplefootballWeb.{Repo, Competition}
+  alias SimplefootballWebWeb.{CompetitionView, MatchdayView}
+
+  require Logger
+
+  def index(conn, _params) do
+    data = competitions()
+    json(conn, CompetitionView.renderList(data))
+  end
+
+  def current_matchday(conn, assigns) do
+    competition = competitionByType(assigns["competitionType"])
```

Comment:
Kann es sein, dass man in einem solchen Fall in Elixir eher irgendwie pattern matching benutzt, um den competetitionType aus den "assigns" rauszubekommen?

Ohne jetzt mega viel zu suchen passt vielleicht das hier: https://elixirforum.com/t/pattern-matching-params/12936/3